README.md

```diff
@@ -13,6 +13,22 @@ Any gotchas that aren’t intuitive.
 
 We require specific versions of Node (8.11.4) and Ruby (2.5.3) in this project. In order to get the correct versions installed, it is recommened to use a VM (version manager) like [asdf-vm](https://asdf-vm.com/#/).
 
+### Lodash
+
+This project uses [`lodash`](https://lodash.com/docs/4.17.11) to provide some additional JavaScript helper functions. Check out lodash's developer documentation, it's super helpful.
```

Comment:
Slight nitpick: it looks like we're installing `^4.17.15` in package.json, but this links to the 4.17.11 docs. I can't imagine there are significant API changes between those two releases, but maybe best to just link to `docs/4.17` to be on the safe side?

Reply:
Weird, at the time I submitted this the latest version of the docs was `4.17.11`. But now it just redirects to `4.17.15` 🤷‍♂ .

I'll update it, regardless.